plugins/bulk-import/src/utils/repository-utils.tsx

```diff
@@ -2,67 +2,22 @@ import React from 'react';
 
 import { Link } from '@backstage/core-components';
 
-import ReadyIcon from '@mui/icons-material/CheckOutlined';
 import { get } from 'lodash';
 
-import {
-  AddRepositoriesData,
-  AddRepositoriesFormValues,
-  Order,
-} from '../types';
-
-export const getRepositoryStatus = (
-  status: string,
-  isItemSelected?: boolean,
-  isDrawer: boolean = false,
-  selectedRepositories?: number,
-) => {
-  if (isItemSelected && !isDrawer) {
-    return (
-      <span>
-        {selectedRepositories || ''}
-        <ReadyIcon
-          color="success"
-          style={{ verticalAlign: 'sub', paddingTop: '7px' }}
-        />
-        Ready
-      </span>
-    );
-  }
-  if (status === 'Exists') {
-    return <span style={{ color: 'grey' }}>Repository already added</span>;
-  }
-
-  return !isDrawer ? <span>Not generated</span> : null;
-};
-
-export const getRepositoryStatusForOrg = (
-  data: AddRepositoriesData,
-  alreadyAdded: number,
-) => {
-  const isSelected = data.selectedRepositories && data.selectedRepositories > 0;
-  const allSelected =
-    (data.selectedRepositories || 0) + alreadyAdded ===
-    data.repositories?.length;
-
-  if (!isSelected) {
-    return <span>Not generated</span>;
-  }
-
-  if (allSelected) {
-    return getRepositoryStatus('Exists', true);
-  }
-
-  return getRepositoryStatus('Exists', true);
-};
+import { AddedRepositories, AddRepositoriesData, Order } from '../types';
 
 const descendingComparator = (
   a: AddRepositoriesData,
   b: AddRepositoriesData,
   orderBy: string,
 ) => {
-  const value1 = get(a, orderBy);
-  const value2 = get(b, orderBy);
+  let value1 = get(a, orderBy);
+  let value2 = get(b, orderBy);
+
+  if (orderBy === 'selectedRepositories') {
```

Comment:
do we need the specific check?

Reply:
The `selectedRepositories` value is an array, I am sorting on the basis of the array length